Sources/Node/Commands/AddProcessCommand.cpp

```diff
@@ -3,3 +3,12 @@
 //
 
 #include "AddProcessCommand.h"
+
+AddProcessCommand::AddProcessCommand(Json::Value json) : Command(CommandType::NEW_PROCESS){
+  Json::Value processDetails = json["processDetails"];
+  this->processContent = processDetails["content"].asString()
```

Comment:
Using this-> is not necessary when referring to a class's variables. I don't think it helps readability, but that's up for debate and doesn't really matter all that much ;)